### PR TITLE
Added explaination of content headers

### PR DESCRIPTION
**Pull request by whiskeysierra**

### CODE REVIEW

File: headers/CommonHeaders.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+Content or entity headers are headers with a `Content-` prefix. They describe the content of the body of the message and
+they can be used in both, HTTP requests and responses. Commonly used content headers include but are not limited to:
+
+ - `Content-Disposition`
+ - `Content-Encoding`
+ - `Content-Length`
+ - `Content-Language`
+ - `Content-Location`
+ - `Content-Range`
+ - `Content-Type`
```
Comment (ePaul) [inline headers/CommonHeaders.md]: Not sure if we want to add a short sentence to each, or at least add a link to the defining section of RFC 7230/7231/7233?

File: headers/CommonHeaders.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
 
 This section describes a handful of headers, which we found raised the most questions in our daily usage, or which are useful in particular circumstances but not widely known.
 
+## Content Headers
+
+Content or entity headers are headers with a `Content-` prefix. They describe the content of the body of the message and
+they can be used in both, HTTP requests and responses. Commonly used content headers include but are not limited to:
```
Comment (ePaul) [inline headers/CommonHeaders.md]: Maybe add an » (if they have a body)« after »responses«?

Comment (whiskeysierra) [inline headers/CommonHeaders.md]: Seems redundant to me.

File: headers/CommonHeaders.md (hunk 1 above)
Comment (whiskeysierra) [inline headers/CommonHeaders.md]: Can you modify the file yourself?

Comment (ePaul) [inline headers/CommonHeaders.md]: Can do, this was more of a question if this is desired.

**Comment by fmueller** [comment]
Just an editorial note: Can you bring it into the MUST/SHOULD/COULD-rule form to make it consistent with our guidelines please?

**Comment by whiskeysierra** [comment]
> Just an editorial note: Can you bring it into the MUST/SHOULD/COULD-rule form to make it consistent with our guidelines please?

It's not really a rule, more a guideline :stuck_out_tongue_closed_eyes:

**Comment by fmueller** [comment]
Well, can you transform it for each header into a could-rule?

File: headers/CommonHeaders.md (hunk 1 above)
Comment (ePaul) [inline headers/CommonHeaders.md]: Added, feel free to edit, revert or reset.

**Comment by tkrop** [comment]
:+1:

**Comment by whiskeysierra** [comment]
:+1:

**Pull request merged**